mne_bids_pipeline/_config.py

```diff
@@ -666,6 +666,9 @@
    from mne.transforms import translation
    mf_destination = translation(z=0.04)
    ```
+3. Compute the time-weighted average head position for each run, and use that as the
```

Comment:
Just to make it clearer
```suggestion
3. Compute the time-weighted average head position across all runs, and use that as the
```